cmd/kar-controllers/app/metrics.go

```diff
@@ -0,0 +1,48 @@
+package app
+
+import (
+	"runtime"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
```

Comment:
TBH not 100% sure how useful these all are, they appear to be some of the more common metrics out there on the net, open to opinions on them

Reply:
I think they are good start, we should see in practice how usefull they turned out.  IMHO, in the current emplementation having a view on GC pauses and GOROUTINE counts is necessary. The backoff logic uses deamon go routines to sleep. Under, extreme, extreme cases one could run out of those.